rust/Cargo.toml

```diff
@@ -15,7 +15,7 @@ serde_yaml = "0.9"
 serde = {version = "1.0.137", default-features = false, features = ["derive"]}
 serde_json = { version = "1.0.75", default-features = false }
 nmstate = { path = "src/lib", version = "2.2", default-features = false }
-nispor = "1.2.24"
+nispor = { git = "https://github.com/nispor/nispor.git", rev = "071a2c8c0e406b2248a3082961df7efa057d8aad" }
```

Comment:
Remember to change this to a version number.

Reply:
I have release nispor 1.2.25. You may use it now.